package.json

```diff
@@ -32,76 +32,86 @@
     "flowchart",
     "diagram-maker"
   ],
+  "dependencies": {
+    "classnames": "^2.3.1",
+    "immer": "^9.0.12",
+    "lodash-es": "^4.17.21",
+    "preact": "^10.6.5",
+    "react-redux": "^7.2.6",
+    "redux": "^4.1.2",
+    "redux-undo-redo": "^2.1.1",
+    "uuid": "^8.3.2"
+  },
   "devDependencies": {
-    "@cypress/code-coverage": "^3.9.10",
-    "@types/classnames": "2.2.x",
-    "@types/dagre": "^0.7.46",
-    "@types/jest": "^26.0.24",
-    "@types/lodash-es": "4.17.x",
-    "@types/uuid": "^8.3.1",
-    "@types/webpack": "^4.41.30",
-    "@types/webpack-env": "1.15.x",
-    "autoprefixer": "9.8.x",
-    "classnames": "2.2.x",
-    "css-loader": "4.2.x",
-    "cypress": "5.0.x",
-    "dagre": "^0.8.5",
-    "dts-bundle-generator": "5.4.x",
-    "es6-shim": "0.35.x",
-    "fork-ts-checker-webpack-plugin": "5.1.x",
-    "html-webpack-plugin": "4.3.x",
-    "immer": "9.0.x",
-    "istanbul-instrumenter-loader": "3.0.x",
-    "istanbul-lib-coverage": "3.0.x",
-    "jest": "26.4.x",
-    "jest-snapshot": "26.4.x",
-    "lodash-es": "4.17.x",
-    "mini-css-extract-plugin": "^1.6.2",
-    "node-sass": "^5.0.0",
-    "nyc": "15.x",
-    "optimize-css-assets-webpack-plugin": "^6.0.1",
-    "postcss": "8.2.x",
-    "postcss-loader": "3.0.x",
-    "postcss-scss": "2.1.x",
-    "preact": "8.5.x",
-    "preact-redux": "2.0.x",
-    "preact-render-spy": "1.x",
-    "redux": "4.0.x",
-    "redux-undo-redo": "2.1.x",
-    "sass-loader": "10.0.x",
-    "ts-jest": "26.3.x",
-    "ts-loader": "8.0.x",
-    "tslint": "5.11.x",
-    "tslint-config-airbnb": "5.9.x",
-    "tslint-eslint-rules": "5.3.x",
-    "tslint-react": "3.6.x",
-    "typedoc": "^0.20.37",
-    "typescript": "4.2.x",
-    "uuid": "8.x",
-    "webpack": "^4.44.1",
-    "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.0",
-    "webpack-merge": "5.1.x",
-    "glob-parent": ">=5.1.2",
-    "trim-newlines": ">=3.0.1"
+    "@cypress/code-coverage": "^3.9.12",
+    "@types/dagre": "^0.7.47",
+    "@types/enzyme": "^3.10.11",
+    "@types/jest": "^27.4.0",
+    "@types/lodash-es": "^4.17.6",
+    "@types/uuid": "^8.3.4",
+    "@types/webpack": "^5.28.0",
+    "@types/webpack-env": "^1.16.3",
+    "@typescript-eslint/eslint-plugin": "^5.10.2",
+    "@typescript-eslint/parser": "^5.10.2",
+    "autoprefixer": "^10.4.2",
+    "babel-loader": "^8.2.3",
+    "babel-plugin-istanbul": "^6.1.1",
+    "bundle-declarations-webpack-plugin": "^2.2.2",
+    "css-loader": "^6.6.0",
+    "cypress": "^9.4.1",
+    "enzyme": "^3.11.0",
```

Comment:
I dont think moving to enzyme is going to be the correct long term choice,the maintainer has handed over the project and reccomended people migrate due to it likely never supporting react 18. We should instead consider `react-testing-library` and `testing-library`

Reply:
Good catch. I wasn't aware of this. This would require some more discussion & investigation to replace, so maybe we can take this up as a follow up, given that this PR would also upgrade several dependencies that are already outdated. 

Also worth noting is that we are using enzyme with preact adapter and preact closely follows react versions but isn't 1 to 1 with react. And with preact, the adapter we use is the pure variant, which doesn't rely on react compatibility layer so may be the enzyme compatibility for preact might be longer lived than enzyme compatibility with react.